### ISSUE TITLE
force Spring to produce JSON output

### ISSUE THREAD

**Issue by thombergs** [reflectoring/coderadar#86]
some browsers by default get xml as answer of an REST API call. Spring has to be forced to produce JSON by adding the following:

``` java
@RequestMapping(value = "/{widgetId}", method = GET, produces = MediaType.APPLICATION_JSON_VALUE)
```

**Comment by jo2** [comment]
related: #110 

Those two issues can fixed in the same PR, because they affect the exact same rows.

**Comment by maximAtanasov** [comment]
Fixed in #294
